### ISSUE TITLE
Displaying double negative on percentage decreases

### ISSUE THREAD

**Issue by swade1987** [infracost/infracost#598]
See below ...

```
Monthly cost change for /home/atlantis/.atlantis/repos/XXX.tfplan
    Amount:  -$281 ($914 -> $633)
    Percent: --31%
```
Looks like a bug in this function: https://github.com/infracost/infracost/blob/master/internal/output/diff.go#L281

**Comment by aliscott** [comment]
Fixed in https://github.com/infracost/infracost/pull/636, thanks @swade1987 and @ekifel.

**Comment by alikhajeh1** [comment]
@swade1987 this was shipped in v0.8.7 (we've also upgraded the infracost-atlantis image to the latest version of Atlantis and the Infracost 0.8.7 CLI version)
